storage/byodb/external-buckets/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: External Buckets
+permalink: /storage/byodb/external-buckets/
+---
+
+* TOC
+{:toc}
+
+If you operate Keboola in <abbr title="Bring Your Own Database">BYODB</abbr> mode on top of you own data warehouse, the data residing in the warehouse is not automatically visible or accessible from inside Keboola. We're addressing this by providing external buckets feature.
```

Comment:
ni Bring Your Own Database (BYODB) mode
your own
**External buckets** feature

Reply:
I don't understand your comment here?

Reply:
Typos. 

Bring Your Own Database (BYODB) mode <--- not abbreviated
you**r** own
**E**xternal buckets feature